.travis.yml

```diff
@@ -38,14 +38,13 @@ before_install:
       # TODO: need script for Windows to generate SSL certs...
       # Install Redis
       choco install redis-64 -v
+      choco install openssl.light
+      mkdir certs
+      ./tools/generate-ssl-certs.sh
```

Comment:
Running `.sh` on Windows is cool?  Does it have WSL installed?

Reply:
It has WSL pre-installed, one of my force pushes tested this out, and it showcased that it was "already installed"